gateway/examples/llm-proxy.yaml

```diff
@@ -17,69 +17,78 @@
 # --------------------------------------------------------------------
 
 version: ai.api-platform.wso2.com/v1
-kind: llm/proxy
-data:
+kind: LlmProxy
+spec:
```

Comment:
We need to format the yaml properly. Please refer
https://github.com/wso2/api-platform/discussions/492#discussioncomment-15300717

missing metadata field as well